mysql-test/suite/binlog/r/binlog_gtid_recovery.result

```diff
@@ -0,0 +1,22 @@
+RESET MASTER;
+CREATE TABLE t1 (c1 INT NOT NULL PRIMARY KEY) ENGINE=InnoDB;
+SHOW BINARY LOGS;
+Log_name	File_size
+master-bin.000001	352
+SELECT * FROM mysql.gtid_executed;
+source_uuid	interval_start	interval_end
+Kill the server
+# Kill the server
+Restart the server only to kill it before GTID is saved during recovery process
+Start the server again to see if server recovers GTIDs from binlogs and updates @@global.gtid_executed and mysql.gtid_executed table.
+# restart
+SELECT interval_start, interval_end from mysql.gtid_executed;
+interval_start	interval_end
+1	1
+SHOW BINARY LOGS;
+Log_name	File_size
+master-bin.000001	352
```

Comment:
@VarunNagaraju Could you also check if this test passes for the second time when we run with `--repeat=2`.
I see that we have binlog file names (`000001 ` / `000002` / `000003`) recorded here and want to double check.

Reply:
@percona-ysorokin The test performs `RESET MASTER` in the beginning and it should handle the hardcoded binlog names.

Reply:
Oops, missed that part somehow. Thanks, Venki. Please, ignore.